interface/notes_for_s3_interface_deployment.md

```diff
@@ -0,0 +1,18 @@
+# Notes for deploying interface single page app to AWS S3 bucket
+
+Download and configure aws command line interface 
+(http://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html)
+
+Set up users, passwords and permissions with AWS IAM
+(http://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html)
+
+Initialize S3 bucket using AWS console or something like terraform
+(https://www.terraform.io/)
+
+Upload bin folder to S3 bucket using aws-cli 'sync' command
+
+```shell
+# Inside the current folder (adage-server/interface), run
+aws s3 sync ./bin s3://<your s3 bucket name>
+
+```
```

Comment:
I still don't see much benefit to using an S3 bucket for this (some negatives, in fact: cost, plus extra administrative overhead), but maybe I'm missing something. We don't need to nail down this deployment detail just yet anyway.